apps/betterangels-backend/shelters/enums.py

```diff
@@ -208,19 +218,19 @@ class SPAChoices(models.IntegerChoices):
 
 
 class PetChoices(models.TextChoices):
-    CATS = "Cats", _("Cats")
-    DOGS_OVER_25_LBS = "Dogs >25lbs", _("Dogs >25lbs")
-    DOGS_UNDER_25_LBS = "Dogs <25lbs", _("Dogs <25lbs")
-    EMOTIONAL_SUPPORT = "Emotional Support", _("Emotional Support")
-    EXOTICS = "Exotics", _("Exotics")
-    SERVICE_ANIMAL = "Service Animal", _("Service Animal")
+    CATS = "cats", _("Cats")
+    DOGS_OVER_25_LBS = "dogs_over_25lbs", _("Dogs >25lbs")
+    DOGS_UNDER_25_LBS = "dogs_under_25lbs", _("Dogs <25lbs")
```

Comment:
A small thing, and it may not be relatable since this is mainly admin stuff -- I personally think this could be more explicit as Dogs over 25 lbs and Dogs under 25 lbs.

Thinking it from the lens of the Web Content Accessibility Guidelines (WCAG) 2.1:
> While WCAG does not explicitly state a preference for words over symbols in all contexts, it emphasizes the importance of clear and simple language (Guideline 3.1) and making content understandable (Guideline 3.2). Using words instead of symbols helps ensure that content is clear and understandable, especially for users with cognitive disabilities or those using screen readers.